Build/validate-qir.ps1

```diff
@@ -30,19 +30,33 @@ function Build-One {
         Write-Host "##vso[task.logissue type=error;]Failed to build $Project."
         $script:allOk = $False
     } else {
+        Write-Host "##[info]Compiling $projectName to executable..."
         qir-cli `
             --dll (Join-Path bin $Env:BUILD_CONFIGURATION netcoreapp3.1 "${projectName}.dll") `
             --exe qir
-        if  ($LastExitCode -ne 0) {
-            Write-Host "##vso[task.logissue type=error;]Failed to build $Project."
+        if  ($LastExitCode -ne 0 -or (Get-ChildItem (Join-Path qir *__Interop.exe)).count -eq 0) {
+            Write-Host "##vso[task.logissue type=error;]Failed to compile $projectName to executable."
             $script:allOk = $False
         } else {
+            Write-Host "##[info]Updating environment path variables."
+            $qirPath = Join-Path $projectDirectory qir
+            $old_DYLD_LIBRARY_PATH = $env:DYLD_LIBRARY_PATH
```

Comment:
This looks like it sets library paths for macOS and Linux hosts, is anything needed for Windows hosts?

Reply:
On Windows, the os is smart enough to look in the executable's location for library files, as well as PATH locations. Linux and Mac needed a little more help here, as they don't have that behavior.

Reply:
That behavior is configurable on Linux and macOS, I believe, by setting `rpath` to `$ORIGIN` when compiling the executable.